### PR TITLE
[RLlib] IMPALA on new API stack: Add Pong-v5 release test.

### PR DESCRIPTION
**Pull request by sven1977**
IMPALA on new API stack: Add Pong-v5 release test.

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: rllib/tuned_examples/impala/pong_impala.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                "uses_new_env_runners": True,
+            }
+            if not args.use_tiny_cnn
+            else {}
+        ),
+    )
+)
+
+stop = {
+    "env_runner_results/episode_return_mean": args.stop_reward,
```
Comment (simonsays1980) [inline rllib/tuned_examples/impala/pong_impala.py:88]: Can we use here our constants? Like this we have more unity across our examples which helps users to get some structure.

**Review by simonsays1980 — APPROVED** [review]
LGTM.

File: rllib/tuned_examples/impala/pong_impala.py (hunk 1 above)
Comment (sven1977) [reply rllib/tuned_examples/impala/pong_impala.py:88]: done

**Review by aslonnie — APPROVED** [review]
<div><br/>via <a href="https://giphy.com/firstandmonday/">FirstAndMonday</a> on <a href="https://giphy.com/gifs/firstandmonday-lol-toolongdidntread-didntread-ToMjGpqWojvqz7ts2li">GIPHY</a></div>

**Pull request merged**